src/ansible_navigator/configuration_subsystem/navigator_post_processor.py

```diff
@@ -281,6 +281,20 @@ def execution_environment(self, entry: SettingsEntry, config) -> PostProcessorRe
                         )
                     if hint:
                         exit_messages.append(ExitMessage(message=hint, prefix=ExitPrefix.HINT))
+
+            # check for /dev/mqueue when using podman because runner passes ipc=host
+            # https://github.com/ansible/ansible-navigator/issues/610
+            if config.container_engine == "podman" and not Path("/dev/mqueue").exists():
+                exit_msg = (
+                    "Execution environment support while using podman requires a"
+                    " '/dev/mqueue' directory."
```

Comment:
Same as with the change note
```suggestion
                    " '/dev/mqueue/' directory."
```

Reply:
used a var so it's consistent, thx